docs/management/hosts/hosts-overview.asciidoc

```diff
@@ -6,31 +6,45 @@ The Hosts page provides a comprehensive overview of all hosts and host-related s
 
-NOTE: The default time range is within the last 15 minutes.
+KPI charts show metrics for hosts and unique IPs within the time range specified in the date picker. Data in the KPI charts is visualized through linear or bar graphs.
 
-*Data tables*
+TIP: Hover inside a KPI chart to display the actions menu (*...*), where you can perform these actions: inspect, open in Lens, and add to a new or existing case.
+
+[[host-data-tables]]
+[discrete]
+== Data tables
 
 Beneath the KPI charts are data tables, categorized by individual tabs, which are useful for viewing and investigating specific types of data. Select the relevant tab to view the following data:
 
 * *All hosts*: High-level host details.
 * *Uncommon processes*: Uncommon processes running on hosts.
 * *Anomalies*: Anomalies discovered by machine learning jobs.
 * *Events*: All host events. To display <<det-engine-terminology, alerts>> received from external monitoring tools, scroll down to the Events table and select *Show only external alerts* on the right.
+* *Host risk*: The latest recorded host risk score for each host and its host risk classification. This feature requires a www.elastic.co/pricing[Platinum subscription] or higher and must be enabled to display the data. Click *Enable* on the *Host risk* tab to get started. To learn more, refer to our <<host-risk-score, host risk score documentation>>.  
 * *Sessions*: Linux process events that you can open in <<session-view, Session View>>, an investigation tool that allows you to examine Linux process data at a hierarchal level.
 
 The tables within the *Events* and *Sessions* tabs include inline actions and several customization options. To learn more about what you can do with the data in these tables, refer to <<alerts-ui-manage>>.
 
 [role="screenshot"]
 image::images/events-table.png[Events table]
 
+[[host-details-page]]
+[discrete]
+== Host details page
+
+A host's details page displays all relevant information for the selected host. To view a host's detail page, click its *Host name* link from the *All hosts* table.
 
-*Host detail pages*
+The host details page includes the following sections: 
 
-A host's detail page displays all relevant information for the selected host, such as when the host was first and last seen, associated IP addresses, and operating system. It also displays graphs for unique IPs, destinations, and user authentications. To view a host's detail page, click its *Host name* link from the *All hosts* list.
+* *Summary*: Details such as the user ID, when the user was first and last seen, the associated IP addresses, and operating system. If the user risk score feature is enabled, this section also displays user risk score data. 
+* *Alert metrics*: The total number of alerts by severity, rule, and status (`Open`, `Acknowledged`, or `Closed`).  
+* *Data tables*: The same data tables on the main Users page, except the values are for the selected user instead of all users. 
```

Comment:
```suggestion
* *Data tables*: The same data tables as on the main Users page, except with values for the selected user instead of for all users. 
```